config/initializers/content_security_policy.rb

```diff
@@ -31,10 +31,11 @@
                       :https,
                       :unsafe_inline
 
-  policy.connect_src  :self,
-                      :https,
-                      'http://localhost:3000',
-                      'ws://localhost:3000'
+  if Rails.env.development?
+    policy.connect_src :self, :https, 'http://localhost:3000', 'ws://localhost:3000'
```

Comment:
Recommending we whittle this down to a more restrictive list of sources, such as 
`:self, 'https://*.amplify.gov.au', 'https://*.sl.nsw.gov.au'`, and whatever is needed for external services such as Google,  Twitter, Facebook, Google Analytics.